mvcc/backend/backend.go

```diff
@@ -391,7 +391,9 @@ func (b *backend) defrag() error {
 	err = defragdb(b.db, tmpdb, defragLimit)
 	if err != nil {
 		tmpdb.Close()
-		os.RemoveAll(tmpdb.Path())
+		if rmErr := os.RemoveAll(tmpdb.Path()); rmErr != nil {
```

Comment:
@tedyu looks good to log error but how about adding `plog.Errorf` as well similar to LOC 414 as done for `err = os.Rename(tdbp, dbp)` ? Thanks!

Reply:
I don't seem to find plog.Errorf call in this class.

I have added nil check for b.lg

Reply:
Hope I didn't misunderstand you but b.lg.Error is also a new one that you are adding so what I am suggesting is to cover plog as well similar to other places in here.

Reply:
@spzala 
your comment has been addressed